app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -169,27 +177,190 @@
 
     angular.extend(vm, {
       active: 'warlock',
+      activesets: '5/5/1',
+      progress: 0,
+      allSetTiers: [],
+      highestsets: {},
+      topsets: [],
+      lockeditems: { helmet: null, gauntlets: null, chest: null, leg: null, classItem: null, ghost: null, artifact: null },
       normalize: 335,
       doNormalize: false,
       type: 'Helmets',
       showBlues: false,
       showExotics: true,
       showYear1: false,
       combinations: null,
+      setOrder: '-str_val,-disc_val,-int_val',
+      setOrderValues: ['-str_val', '-disc_val', '-int_val'],
       statOrder: '-stats.STAT_INTELLECT.value',
       ranked: {},
       filter: {
         int: 3,
         dis: 2,
         str: 2
       },
+      lockedItemsValid: function(dropped_id, dropped_type) {
+          dropped_id = dropped_id.split('-')[1];
+          var item = _.findWhere(buckets[vm.active][dropped_type], {id: dropped_id});
+          var exoticCount = ((item.tier === 'Exotic')? 1 : 0);
+          for(var type in vm.lockeditems) {
+              var item = vm.lockeditems[type];
+              if(item === null || type === dropped_type) { continue; }
+              if(item.tier === 'Exotic' && item.type != 'classItem') {
```

Comment:
I think @delphiactual might have been alluding this, but when looking at
the mid-tier options, I'm constantly doing the math in my head to determine
whether it's T10/T11/T12.  Perhaps include the Txx aspect somewhere?
Filtering? In the "5/5/2 (T12)" string? Other?

On Sun, May 22, 2016 at 6:23 PM, ericnelson0 <notifications@github.com>
wrote:

> In app/scripts/minmax/dimMinMax.controller.js
> <https://github.com/DestinyItemManager/DIM/pull/538#discussion_r64160980>:
>
> >        statOrder: '-stats.STAT_INTELLECT.value',
> >        ranked: {},
> >        filter: {
> >          int: 3,
> >          dis: 2,
> >          str: 2
> >        },
> > +      lockedItemsValid: function(dropped_id, dropped_type) {
> > +          dropped_id = dropped_id.split('-')[1];
> > +          var item = _.findWhere(buckets[vm.active][dropped_type], {id: dropped_id});
> > +          var exoticCount = ((item.tier === 'Exotic')? 1 : 0);
> > +          for(var type in vm.lockeditems) {
> > +              var item = vm.lockeditems[type];
> > +              if(item === null || type === dropped_type) { continue; }
> > +              if(item.tier === 'Exotic' && item.type != 'classItem') {
>
> Thanks, I don't have an exotic cape so I haven't been able to test that.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/538/files/59aaaec29b25aa17f2eab064b7d5e6b35c64c833#r64160980>
>